pkgs/applications/office/calligra/default.nix

```diff
@@ -8,7 +8,7 @@
   kwindowsystem, kxmlgui, sonnet, threadweaver,
   kcontacts, akonadi, akonadi-calendar, akonadi-contacts,
   eigen, git, gsl, ilmbase, kproperty, kreport, lcms2, marble, pcre, libgit2, libodfgen,
-  librevenge, libvisio, libwpd, libwpg, libwps, okular, openexr, openjpeg, phonon,
```

Comment:
Another nit: The commit for calligra should explain that openjpeg is used / not used / isn't mandatory for it's build? And that the latest poppler can be used and poppler_0_61 is not needed.